### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DEBSUBHRO**
Added and changed some terms for a better view of the document.
Removed the bulletins from the "currently supported platforms" section for a better view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <p align="center">
-    <a href="https://appwrite.io" target="_blank"><img width="260" height="39" src="https://appwrite.io/images/github-logo.png" alt="Appwrite Logo"></a>
+    <a href="https://appwrite.io" target="_blank"><img width="260px" height="39px" src="https://appwrite.io/images/github-logo.png" alt="Appwrite Logo"></a>
```
Comment (eldadfux) [inline README.md]: No need for px when used as an HTML attribute, only when used in CSS

Comment (DEBSUBHRO) [reply README.md]: > No need for px when used as an HTML attribute, only when used in CSS

Okay sir. Thank you.

**Comment by eldadfux** [comment]
@DEBSUBHRO thank you for your help! Please reach out on Gitter if you would like me to send you some Appwrite stickers and magnets.

**Comment by eldadfux** [comment]
@DEBSUBHRO Feel free to add your name in the README credits 👍🏻

**Pull request merged**